pkg/openevec/config.go

```diff
@@ -210,6 +211,7 @@ type PodConfig struct {
 	OpenStackMetadata bool
 	DatastoreOverride string
 	ACLOnlyHost       bool
+	VirtMode          config.VmMode
```

Comment:
Would be great to add related `virt-mode` flag into [podDeployCmd](https://github.com/lf-edge/eden/blob/f81d9918ecdb5532c14f59ae4f4db7603cc74b0a/cmd/edenPod.go#L126) to support it in CLI.

Reply:
sure!

Reply:
Added, is the default value is correct? or should be something else?

Reply:
I am not sure, please [check](https://github.com/lf-edge/eden/blob/f81d9918ecdb5532c14f59ae4f4db7603cc74b0a/pkg/expect/expectation.go#L142-L149).

Reply:
I set the default according to what you provided.

Reply:
Looks like some tests failed with `level=fatal msg="Unknown virt-mode: "` (virtualization test). Can you please try to deploy something locally?

Reply:
shoot, sorry.